site/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { graphql, Link, useStaticQuery } from 'gatsby'
+
+import * as styles from './Header.module.css'
+import { PropsWithChildren } from 'react'
+
+
+type QueryData = {
```

Comment:
Can we create file for each type? or move types to different file

Reply:
will be done in #3